ludwig/features/category_feature.py

```diff
@@ -394,12 +393,7 @@ def postprocess_predictions(
         postprocessed = {}
         name = self.feature_name
 
-        npy_filename = None
-        if is_on_master():
```

Comment:
here the logic was that each worker was actually doing postprocessing so they would override each other's outputs if they were on the same machine (like one machine with multiple gpus). I'm not sure about how this issue is addressed now.

Reply:
Replicated for all output features

Reply:
This is handled in the `postprocess` function which calls into these output feature functions here: https://github.com/ludwig-ai/ludwig/blob/backend_refactor/ludwig/data/postprocessing.py#L35

This saves us from having to pass around the `backend` to all these features just for checking this one condition, and removes the duplicate code.